lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { DatePicker } from '../DatePicker';
+import { TextField } from '@material-ui/core';
+import { screen } from '@testing-library/react';
+import { createClientRender } from './createClientRender';
+
+describe('<DatePicker />', () => {
+  const render = createClientRender({ strict: false });
+  it('Allows to select edge years from list', () => {
```

Comment:
```suggestion
  // StrictModeViolation: uses Grow
  const render = createClientRender({ strict: false });
  
  it('should disable the year on the edge', () => {
```

Reply:
Oh my gosh, just realize that this test will fail each year on January 1st. I hate writing tests for datepickers XD